GPT/modelInsertionMethod.talon-list

```diff
@@ -0,0 +1,10 @@
+list:  user.modelInsertionMethod
+
+-
+
+above
+below
+clipped
+selected
```

Comment:
Let me know if these seem reasonable. There is always a tradeoff in wording since some of these are locations and some of them are moreso actions (i.e. selecting), but I was hoping to have them all fit syntactically in the same commandgrammar. 

I am using 'above' and 'below' since 'after' in my opinion isn't as clear since it could imply something temporal.